cloud/ec2_client.go

```diff
@@ -1349,7 +1349,7 @@ func (c *awsClientMock) ChangeResourceRecordSets(ctx context.Context, input *rou
 func (c *awsClientMock) AssumeRole(ctx context.Context, input *sts.AssumeRoleInput) (*sts.AssumeRoleOutput, error) {
 	c.AssumeRoleInput = input
 	if input.DurationSeconds == nil {
-		input.DurationSeconds = aws.Int32(int32(time.Hour.Seconds()))
+		input.DurationSeconds = aws.Int32(int32(time.Hour.Seconds() / 4))
```

Comment:
The default in AWS documentation is 15m, so I adjusted this just to be more semantic